interfaces/Tuit2TagControllerI.ts

```diff
@@ -3,5 +3,7 @@ import {Request, Response} from "express";
 export default interface Tuit2TagControllerI {
     createTuit2Tag(req: Request, res: Response): void;
     deleteTuit2Tag(req: Request, res: Response): void;
-    findTuit2TagsByTuit(req: Request, res: Response): void;
```

Comment:
I think I need this method still

Reply:
Did you end up using findTuit2TagsByTuit or findTagsByTuit? I'm guessing you'd need it in the DAO?

Reply:
I use the Dao version.  Let's keep the controller version for testing purposes.  It may not get used for practical applications though.

Reply:
I'd have to reformat the link if we keep it in the controller so it doesn't cause issues with having the same path as findTagsByTuit:
            app.get("/api/tuits/:tuitID/tags", Tuit2TagController.tuit2tagController.findTuit2TagsByTuit);
            app.get("/api/tuits/:tuitID/tags", Tuit2TagController.tuit2tagController.findTagsByTuit);

If you want to keep findTuit2TagsByTuit in the controller, maybe we can use "/api/tuit2tags/tuits/:tuitID"?

Reply:
Let's do the first one.  I added a /api/tuit2tags path, so that makes more sense now.